utils/notification_service.py

```diff
@@ -819,8 +819,8 @@ def prepare_reports(title, header, reports, to_truncate=True):
                 stacktraces = handle_stacktraces(artifact["failures_line"])
 
                 for line in artifact["summary_short"].split("\n"):
-                    if re.search("FAILED", line):
-                        line = line.replace("FAILED ", "")
+                    if line.startswith("FAILED "):
```

Comment:
To ensure the 1-1 correspondence between `summary_short.txt` and `failures_line.txt` (see PR description).